frontend/src/components/pages/RegistrantExperience/SessionSelection/SessionCard.tsx

```diff
@@ -0,0 +1,150 @@
+import { SessionCardState } from "./SessionSelectionTypes";
+import {
+  sessionCardBoldTextStyles,
+  sessionCardDatesTextStyles,
+  sessionCardDetailsTextStyles,
+} from "./SessionSelectionStyles";
+import { adjustTimeToAmPm } from "../../../../utils/CampUtils";
+
+const WAITLISTED_COLOR = "red.100";
+const AVAILABLE_COLOR = "green.100";
+
+type SessionCardDetailsProps = {
+  fee: number;
+  startTime: string;
+  endTime: string;
+  campSession: CampSessionResponse;
+};
+
+const SessionCardDetails = ({
+  fee,
+  startTime,
+  endTime,
+  campSession,
+}: SessionCardDetailsProps): React.ReactElement => {
+  const getCampSessionDates = () => {
+    const startDate = new Date(campSession.dates[0]);
+    const endDate = new Date(campSession.dates[campSession.dates.length - 1]);
+    return `${startDate.toLocaleString("default", {
+      month: "short",
+    })} ${startDate.getDay()} - ${endDate.toLocaleString("default", {
+      month: "short",
+    })} ${endDate.getDay()}, ${endDate.getFullYear()}`;
+  };
+
+  const formattedTimeString = (start: string, end: string): string => {
+    return `${adjustTimeToAmPm(start)} - ${adjustTimeToAmPm(end)}`;
+  };
+
+  const [isMobile] = useMediaQuery("(max-width: 767px)");
+  return (
+    <>
+      <Text textStyle={sessionCardDatesTextStyles}>
+        {getCampSessionDates()}
+      </Text>
+      {isMobile ? (
+        <>
+          <Text textStyle={sessionCardDetailsTextStyles}>
+            {formattedTimeString(startTime, endTime)}
+          </Text>
+          <Text textStyle={sessionCardDetailsTextStyles}>${fee}</Text>
+        </>
+      ) : (
+        <Text textStyle={sessionCardDetailsTextStyles}>
+          {formattedTimeString(startTime, endTime)} · ${fee}
+        </Text>
+      )}
+    </>
+  );
+};
+
+export type SessionCardProps = {
+  fee: number;
+  startTime: string;
+  endTime: string;
+  campSession: CampSessionResponse;
+  handleClick: (sessionID: string) => void;
+  state: SessionCardState;
+  sessionIsWaitlisted: boolean;
+};
+
+const SessionCard = ({
+  fee,
+  startTime,
+  endTime,
+  campSession,
+  handleClick,
+  state,
+  sessionIsWaitlisted,
+}: SessionCardProps): React.ReactElement => {
+  const handleCardClick = () => {
+    if (state !== SessionCardState.Disabled) {
+      handleClick(campSession.id);
+    }
+  };
+
+  let backgroundColour = "white";
+
+  if (state === SessionCardState.Selected) {
+    backgroundColour = sessionIsWaitlisted ? WAITLISTED_COLOR : AVAILABLE_COLOR;
+  }
+
+  return (
+    <Box
+      bg={backgroundColour}
+      my={2}
+      p={4}
+      borderColor={
+        sessionIsWaitlisted ? "text.critical.100" : "primary.green.100"
+      }
+      borderWidth="1px"
+      borderRadius="20px"
+      onClick={handleCardClick}
+      _hover={{
+        background:
+          backgroundColour !== "white" || state === SessionCardState.Disabled
+            ? backgroundColour
+            : "RGBA(0, 0, 0, 0.16)",
```

Comment:
should we make this a constant as UNSELECTED_COLOR ?

Reply:
yep, called it HOVERED_COLOR